scripts/mobile_generator.py

```diff
@@ -272,7 +274,9 @@ def __lt__(self, other):
     members.sort()
     
     # Compute output
-    if args.output_language == "swift":
-        print(compute_swift())
+    if args.output_language == "kotlin":
+        print(compute_kotlin(members, enums, event_name))
+    elif args.output_language == "swift":
+        print(compute_swift(members, enums, event_name))
     else:
-        print(compute_kotlin())
+        raise Exception(f"Invalid language option: {args.output_language}")
```

Comment:
OOI isn't `choices=['kotlin', 'swift']` enough to validate the args?

Reply:
Yes it is. I only ended up leaving this in so that if someone adds another language to the arguments and there was a typo in the `if` then it would be more obvious - although I realise how unlikely this is 😅

Reply:
I've updated the message so it's a bit clearer :)